app/views/tournaments/_deck_selection.html.slim

```diff
@@ -0,0 +1,25 @@
+.col-md-6
+  .card
+    .card-header
+      | Please select from your decks below.
+      a< href="https://netrunnerdb.com/en/decks" target="_blank" Set them in NetrunnerDB.
+    ul.list-group.list-group-flush#nrdb_decks_selected style='border-bottom: 0'
+      li.list-group-item.selected-deck
+        .selected-deck-identity style="background-image:url(https://static.nrdbassets.com/v1/small/00006.jpg)"
+        p.mb-1 No runner selected
+      li.list-group-item.selected-deck
+        .selected-deck-identity style="background-image:url(https://static.nrdbassets.com/v1/small/00005.jpg)"
+        p.mb-1 No corp selected
+    ul.list-group.list-group-flush.overflow-auto#nrdb_decks style='height: 24em'
+      - @decks.each do |deck|
```

Comment:
How about a link out to the deck view (in a new window) for each deck as well?

Reply:
I feel like it might be better to handle that in #115 when we can see how it relates to how the deck will display in Cobra. We need to somehow make clear the distinction between what's registered in Cobra vs. what's shown/edited in NRDB.